app/scripts/views/mixins/signin-mixin.js

```diff
@@ -44,6 +43,8 @@ define(function (require, exports, module) {
 
       return this.invokeBrokerMethod('beforeSignIn', account)
         .then(() => {
+          this.logFlowEvent('attempt', 'signin');
```

Comment:
Since you are hard coding the viewName here, is [signInSubmitContext](https://github.com/mozilla/fxa-content-server/pull/4317/files#diff-c77e5dc059d1a6ffd3eaa534e8844820R18) needed on the prototype?